packages/arb-token-bridge-ui/src/components/TransferPanel/TokenImportDialog.tsx

```diff
@@ -358,88 +353,44 @@ export function TokenImportDialog({
       }}
       actionButtonTitle="Import token"
     >
-      <div className="flex flex-col space-y-2 md:min-w-[628px] md:max-w-[628px]">
+      <div className="flex flex-col space-y-4 pt-4">
         {status === ImportStatus.KNOWN && (
-          <span>This token is on an imported token list as:</span>
+          <span>This token is on an imported token list:</span>
         )}
 
         {status === ImportStatus.KNOWN_UNIMPORTED && (
           <span>
             This token hasn&apos;t been imported yet but appears on a token
-            list. Are you sure you want to import it?
+            list:
           </span>
         )}
 
-        {status === ImportStatus.UNKNOWN && (
-          <div className="flex flex-col items-center space-y-3 sm:flex-row sm:space-x-3 sm:space-y-0">
-            <ExclamationCircleIcon
-              style={{ color: '#CD0000' }}
-              className="h-6 w-6"
-            />
-            <div className="flex flex-col">
-              <span>
-                This token isn&apos;t found on an active token list.
-                <br />
-                Make sure you trust the source that led you here.
-              </span>
-            </div>
-          </div>
-        )}
-
-        <div className="flex flex-col items-center py-4">
-          {tokenToImport?.logoURI && (
-            <SafeImage
-              style={{ width: '25px', height: '25px' }}
-              className="mb-2 rounded-full"
-              src={tokenToImport?.logoURI}
-              alt={`${tokenToImport?.name} logo`}
-            />
-          )}
-          <span className="text-xl font-bold leading-6">
-            {tokenToImport?.symbol}
-          </span>
-          <span className="mb-3 mt-0">{tokenToImport?.name}</span>
-          <a
-            href={`${getExplorerUrl(parentChain.id)}/token/${
-              tokenToImport?.address
-            }`}
-            target="_blank"
-            rel="noopener noreferrer"
-            style={{ color: '#1366C1' }}
-            className="break-all underline"
-          >
-            {tokenToImport?.address}
-          </a>
+        <div className="flex flex-col pb-4">
+          <TokenInfo token={tokenToImport} />
```

Comment:
the logo is missing here
<img width="279" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/13184582/587886c3-2f1c-4f4c-9fde-6c2ac141d7dd">
but it's on the token search panel
<img width="500" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/13184582/cacbbf68-c54c-4086-aaa4-4d1d708d8856">